drivers/mongodb/mongodb_test.go

```diff
@@ -0,0 +1,43 @@
+package mongodb
+
+import (
+	"context"
+	"testing"
+
+	"github.com/k1LoW/tbls/schema"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// it is expected to have running https://hub.docker.com/r/weshigbee/docker-mongo-sample-datasets
+func TestAnalyze(t *testing.T) {
+	ctx := context.Background()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI("mongodb://localhost:27017"))
+	if err != nil {
+		t.Errorf("%v", err)
+	}
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+	}()
+	s := &schema.Schema{
+		Name: "MongoDB local `docker-mongo-sample-datasets`",
```

Comment:
I am concerned that the original dataset on https://hub.docker.com/r/weshigbee/docker-mongo-sample-datasets is [404](https://www.mongodb.com/docs/getting-started/shell/import-data/).

Can you find the original dataset?
Or, could you configure it to populate an empty MongoDB like DynamoDB?
https://github.com/k1LoW/tbls/blob/main/testdata/ddl/dynamodb.sh

Reply:
fixed in [f9f7cfb](https://github.com/k1LoW/tbls/pull/339/commits/f9f7cfb275094a68b1cbc2fee717f6f491097079)
removed using of https://hub.docker.com/r/weshigbee/docker-mongo-sample-datasets